kernel/private/classes/ezpkernelweb.php

```diff
@@ -210,25 +210,33 @@ function()
             }
         );
 
-        eZExecution::addFatalErrorHandler(
-            function()
-            {
-                header("HTTP/1.1 500 Internal Server Error");
-                echo "<b>Fatal error</b>: The web server did not finish its request<br/>";
-                if ( ini_get('display_errors') == 1 )
+        $errorINI = eZINI::instance( 'error.ini' );
+        if( $errorINI->hasVariable( 'ErrorSettings-kernel', 'FatalErrorHandler' ) && is_callable( $errorINI->variable( 'ErrorSettings-kernel', 'FatalErrorHandler' ) ) )
```

Comment:
Hi Carlos, I am not sure about what you mean, there is no trailing spaces there.